package.json

```diff
@@ -88,7 +89,7 @@
 		"ytpl": "^2.2.3"
 	},
 	"devDependencies": {
-		"electron": "^12.2.2",
+		"electron": "^16.0.2",
```

Comment:
This seems to break the downloader:
```
Error: unrecognized flag --experimental-wasm-bulk-memory
Try --help for options
[74932:1214/212918.545255:ERROR:node_bindings.cc(276)] Most NODE_OPTIONs are not supported in packaged apps. See documentation for more details.
```
And `SharedArrayBuffer is not defined` when converting - it should be fixable but might need additional work, maybe we can split the PR between fixing the user agents and the electron update?

Reply:
baeebd1 Fixed both of those issues and 1 more :)